packages/block-editor/src/components/list-view/use-list-view-images.js

```diff
@@ -74,8 +75,26 @@ export default function useListViewImages( { clientId, isExpanded } ) {
 		},
 		[ clientId ]
 	);
-
 	const images = useMemo( () => {
+		const blockImageURL = getBlockImage(
+			block.name,
+			block.attributes,
+			'list-view'
+		);
+		if ( blockImageURL ) {
+			return [
+				{
+					url: blockImageURL,
+					alt:
+						block.attributes?.alt ||
+						block.attributes?.mediaAlt ||
+						'',
+					clientId: block.clientId,
+				},
```

Comment:
The `alt` is not currently used. I included it to match `getImage`. Perhaps it can be useful in the future.